tests/phpunit/bootstrap.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Bootstraps unit-tests.
+ *
+ * @package GlotPress
+ * @subpackage Tests
+ */
+
+if ( ! defined( 'GP_TESTS_DIR' ) ) {
+	define( 'GP_TESTS_DIR', __DIR__ );
+}
+
+if ( ! defined( 'GP_DIR_TESTDATA' ) ) {
+	define( 'GP_DIR_TESTDATA', GP_TESTS_DIR . '/data' );
+}
+
+if ( ! defined( 'GP_TESTS_PERMALINK_STRUCTURE' ) ) {
+	define( 'GP_TESTS_PERMALINK_STRUCTURE', '/%postname%' );
+}
+
+if ( ! defined( 'GP_TESTS_PERMALINK_STRUCTURE_WITH_TRAILING_SLASH' ) ) {
+	define( 'GP_TESTS_PERMALINK_STRUCTURE_WITH_TRAILING_SLASH', '/%postname%/' );
+}
+
+$_tests_dir = getenv( 'WP_TESTS_DIR' );
+
+if ( ! $_tests_dir ) {
+	$_tests_dir = rtrim( sys_get_temp_dir(), '/\\' );
+}
+$_tests_dir .= '/wordpress-tests-lib';
+
+if ( ! file_exists( "{$_tests_dir}/includes/functions.php" ) ) {
+	echo "Could not find {$_tests_dir}/includes/functions.php, have you run bin/install-wp-tests.sh ?" . PHP_EOL; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	exit( 1 );
+}
+
+// Give access to tests_add_filter() function.
+require_once "{$_tests_dir}/includes/functions.php";
+
+if ( getenv( 'GLOTPRESS_PATH' ) ) {
+	define( 'EXTERNAL_GP_PATH', getenv( 'GLOTPRESS_PATH' ) );
```

Comment:
We should ensure that this has a trailing slash since we assume it later:
```suggestion
	define( 'EXTERNAL_GP_PATH', trailingslashit( getenv( 'GLOTPRESS_PATH' ) ) );
```